### DISCUSSION TITLE
[Feature Request]:

### DISCUSSION THREAD

**Discussion by zjrwtx** [acheong08/ChatGPT#944] [General]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What would your feature do ?

I'm not familiar with asynchronous development, can you give an example of using an asynchronous api 

### Proposed workflow

1. Go to ....
2. Press ....
3. ...

**Comment by acheong08** [comment]
Sync is preferred. As for async, much of it is trail and error. I haven't got a proper example. Moving this to discussions
